### ISSUE TITLE
Consistency in state naming

### ISSUE THREAD

**Issue by ankitchouhan1020** [amodm/api-covid19-in#5]
Name of a state in data[9].regional.loc is mentioned as Chattisgarh and same state in data[10].regional.loc is mentioned as Chhattisgarh. Maybe you are scrapping the official health ministry website. Please resolve the issue.

**Comment by amodm** [comment]
Yeah, scraping is the reason behind it. I've fixed it for now.

**Comment by nuwandavek** [comment]
Hey! I was using this awesome API for a [dashboard](https://numbersofindia.com/stories/covid19/). I noticed that there are more variations in the names of entities, which can break existing dashboards, especially ones that map this to an existing geojson for spatial plotting. 

Examples

- Pondicherry and Puducherry are both listed in the ['daily api'](https://api.rootnet.in/covid19-in/stats/daily)
- The names of J&K and Ladakh were changed in the ['latest api'](https://api.rootnet.in/covid19-in/stats/latest), and are inconsistent with the daily api
